common/devdoc/gballoc_ll_win32heap.md

```diff
@@ -0,0 +1,107 @@
+# gballoc_ll_win32heap requirements
+================
+
+## Overview
+
+gballoc_ll_win32heap is a module that delegates all call of its APIs to the ones from windows heap functions.
+
+## References
+[Heap Functions](https://docs.microsoft.com/en-us/windows/win32/memory/heap-functions)
+
+
+## Exposed API
+
+```c
+    MOCKABLE_FUNCTION(, int, gballoc_ll_init, void*, params);
+    MOCKABLE_FUNCTION(, void, gballoc_ll_deinit);
+
+    MOCKABLE_FUNCTION(, void*, gballoc_ll_malloc, size_t, size);
+    MOCKABLE_FUNCTION(, void, gballoc_ll_free, void*, ptr);
+    MOCKABLE_FUNCTION(, void*, gballoc_ll_calloc, size_t, nmemb, size_t, size);
+    MOCKABLE_FUNCTION(, void*, gballoc_ll_realloc, void*, ptr, size_t, size);
+```
+
+### gballoc_ll_init
+```c
+MOCKABLE_FUNCTION(, int, gballoc_ll_init, void*, params);
+```
+
+`gballoc_ll_init` globally initializes the module. `params` exists as a placeholder and is ignored.
+
+**SRS_GBALLOC_LL_WIN32HEAP_02_001: [** `gballoc_ll_init` shall call `CreateHeap(0,0,0)`. **]**
+
+**SRS_GBALLOC_LL_WIN32HEAP_02_002: [** `gballoc_ll_init` shall succeed and return 0. **]**
+
+**SRS_GBALLOC_LL_WIN32HEAP_02_003: [** If `CreateHeap` fails then `gballoc_ll_init` shall fail and return a non-0 value. **]**
+
+### gballoc_ll_deinit
+```c
+MOCKABLE_FUNCTION(, void, gballoc_ll_deinit);
+```
+
+`gballoc_ll_deinit` deinitializes the global state and frees all the used resources.
+
+**SRS_GBALLOC_LL_WIN32HEAP_02_016: [** If the global state is not initialized then `gballoc_ll_deinit` shall return. **]**

```

Comment:
>initialized [](start = 66, length = 11)

spec for "deinitialization of global state" missing.

Reply:
Becomes clear with the addition of the global variable explictly stated.